eng/fv/approaches.rst

```diff
@@ -0,0 +1,191 @@
+.. SPDX-License-Identifier: CC-BY-SA-4.0
+
+.. Copyright (C) 2022 Trinity College Dublin
+
+Formal Verification Approaches
+==============================
+
+We present here an overview of a range of formal methods and tools
+that look feasible for use with RTEMS.
+
+A key criterion for any proposed tool is the ability to deploy it
+in a highly automated manner.
+This amounts to the tool having a command-line interface that covers
+all the features we require.
+One such feature is that the tool generates output that can be
+easily transformed into the formats
+used as part of the qualification datapack reports.
+Tools with GUI interfaces can be very helpful while developing
+and deploying formal models, as long as the models/tests/proofs
+can be re-run automatically via the command-line.
+
+Other important criteria concern the support available
+for for test generation support,
+and how close we can connect the formalism to actual C code.
+
+The final key criteria is whatever techniques we propose should fit in 
+with the RTEMS Project Mission Statement (Section 2 in this document).
```

Comment:
better to refer explicitly to the section name and document name, in case this ever moves, or that ever moves.

Reply:
ok